libs/pipelines/track_online_inertial.cpp

```
@@ -538,46 +553,50 @@ bool SolverSfMInertial::solveNextFrame(int64_t time_ns, const sof::FrameState& f
-        landmarks.insert({track_id, track});
-      }
+    map_.get_recent_landmarks(recent_landmarks_);
+    pnp_landmarks_.clear();
+    pnp_landmarks_.reserve(recent_landmarks_.size());
+    for (const auto& [track_id, point_3d] : recent_landmarks_) {
+      Track track;
+      track.setLocation3D(point_3d, TrackState::kTriangulated);
+      pnp_landmarks_.insert({track_id, track});
     }
 
     if (imu_state == StateMachine::State::Ok) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)